librz/type/parser/types_parser.c

```diff
@@ -370,6 +381,7 @@ int parse_struct_node(CParserState *state, TSNode node, const char *text, Parser
 		TSNode first_leaf = ts_node_named_child(child, 0);
 		if (ts_node_is_null(first_leaf)) {
 			node_malformed_error(state, child, text, "field_declaration");
+			free(name);
```

Comment:
I think all these `free(name)` mean we should switch to `goto err` instead.

Reply:
Done.